README.md

```diff
@@ -1,2 +1,82 @@
 # queries
 Hosts reusable log queries which are built into a single queries.json file.
+
+
+# Query database structure
+
+```yaml
+queries:
+  query1:  # that is query id (sorted by id)
+    name: foo  # required
+    regex: bar  # optional
+    pattern: aaa # optional (either regex or pattern required)
```

Comment:
maybe worth to rename "pattern" to "string", if we have regex patterns as well

Reply:
I was considering using `glob` but i am not yet sure if this is supported.

I kinda find the `string` name very confusing.

On the other hand, in order to remove the `Union[regex,pattern]` which adds undesired complexity, how about using pattern, and making `regex: Optional[bool] = false`?

This would make it easy as we can make `pattern` mandatory, and procress it as regex if regex is set , or plain text if `regex: true`?